ex7.17_NumPy_compute_median_flattened.py

```diff
@@ -0,0 +1,15 @@
+import numpy as np
+
+
+def compute_median_flattened():
+    """
+    function that compute the median of flattened given array
```

Comment:
In this case there is no "given" array you created it inside the function. Turn the array into an argument.